docs/installation.rst

```diff
@@ -17,62 +52,134 @@ Pysat itself may be installed from a terminal command line via::
 
    pip install pysat
 
-Or alternatively through github::
+Note that pysat requires a number of packages for the install.  The upper caps
+for packages below have been removed for the upcoming pysat 3.0.0 release.
+The development version can be found in the ``develop-3``
+branch (see above for caveats).
+
+     * beautifulsoup4
+     * h5py
+     * lxml
+     * madrigalWeb
+     * matplotlib
+     * netCDF4
+     * numpy (>=1.12)
+     * pandas (>=0.23, <0.25)
+     * PyForecastTools
+     * pysatCDF
+     * requests
+     * scipy
+     * xarray (<0.15)
+
+The upper caps for packages above will be removed for the upcoming pysat
+3.0.0 release.
+
+**Development Installation**
+
+----
+
+pysat may also be installed directly from the source repository on github::
 
    git clone https://github.com/pysat/pysat.git
    cd pysat
    python setup.py install
 
-An advantage to installing through github is access to the development branches.  The latest bugfixes can be found in the ``develop`` branch.   However, this branch is not stable (as the name implies). We recommend using this branch in a virtual environment or using `python setup.py develop`.::
+An advantage to installing through github is access to the development branches.
+The latest bugfixes can be found in the ``develop`` branch. However, this
+branch is not stable (as the name implies). We recommend using this branch in a
+virtual environment and using::
 
    git clone https://github.com/pysat/pysat.git
    cd pysat
    git checkout develop
    python setup.py develop
 
+The use of `develop` rather than `install` installs the code 'in-place', so
+any changes to the software do not have to be reinstalled to take effect.
 
-Note that pysat requires a number of packages for the install.
+The development version for v3.0 can be found in the ``develop-3``
+branch (see above for caveats).
 
-    * netCDF4
-    * numpy (>=1.12)
-    * pandas (>=0.23)
-    * scipy
-    * xarray
 
-
-**Set Data Directory**
+**Windows**
 
 ----
 
-Pysat will maintain organization of data from various platforms. Upon the first
+To get pysat installed in Windows you need a POSIX compatible C/ Fortran
+compiling environment. This is required to compile the
+`pysatCDF <https://github.com/pysat/pysatCDF/>`_ package.
 
-.. code:: python
+Python environment: Python 2.7.x
```

Comment:
This should be updated.

Reply:
I passed a note to @mddepew who is the original author of that text to see if there are any updates needed for Python 3 on windows, other than that version text there.

Reply:
Updated.